identity_iota_core/src/rebased/sui/move_calls/identity/create.rs

```diff
@@ -67,6 +67,14 @@ where
       vec![ids, vps],
     )
   };
+  // TODO: modify the Rust APIs to account for this.
```

Comment:
Are there open todos here? ^^

If yes, maybe add a task/issue for this and remove the comment? :D